packages/amplify_authenticator/lib/src/blocs/auth/auth_bloc.dart

```diff
@@ -44,10 +45,27 @@ class StateMachineBloc {
   }
 
   //Exception Controller
-  final _exceptionController =
-      StreamController<AuthenticatorException>.broadcast();
+  final StreamController<AuthenticatorException?> _exceptionController =
+      StreamController<AuthenticatorException?>.broadcast();
 
-  Stream<AuthenticatorException> get exceptions => _exceptionController.stream;
+  Stream<AuthenticatorException?> get exceptions => _exceptionController.stream;
+
+  void clearException() {
+    _exceptionController.add(null);
+  }
+
+  Future<void> resendSignUpCode(String username) async {
```

Comment:
Could we move this down with the other service calls? Also, why is it public why the others are private (prefixed with "_")

Reply:
They are public because they don't form part of the state machine which takes events and then triggers those private methods.